### ISSUE TITLE
Support for displays without DC pin?

### ISSUE THREAD

**Issue by Rajssss** [lvgl/lvgl_esp32_drivers#79]
In menuconfig, there is a option to enable or disable the DC. However I beleive none of the drivers implemented it. Having such functionality will improve the usability of the driver, saving one more pin, and support Displays modules with no DC pin exposed.
Even though, I had hard time finding a driver which works without a DC pin, but found some hints to a 9-Bit SPI mode?

**Comment by C47D** [comment]
Hi,

I think you have the ST7735S display, isn't? Can you test this branch? https://github.com/lvgl/lvgl_esp32_drivers/tree/feat/support_no_dc

**Comment by Rajssss** [comment]
I will and let you know.

**Comment by C47D** [comment]
The changes in that branch doesn't take into account the fact that the DC signal tells the display if the bytes it's getting from the MCU are a command or data.

**Comment by Rajssss** [comment]
@C47D sorry it isn't working, getting garbage on the screen.

**Comment by C47D** [comment]
I guess that's expected, the DC signals lets the display know if the data is a data or a command, what display are you working on? Could you share me an image of the connection?

**Comment by Rajssss** [comment]
> I guess that's expected, the DC signals lets the display know if the data is a data or a command, what display are you working on? Could you share me an image of the connection?

Yes, but with DC pin selected it works fine as expected.
Here the display I have: http://www.lcdwiki.com/1.44inch_SPI_Module_ST7735S_SKU:MSP1443
And this is the pinout I am using.

![photo_2021-07-02_08-40-18](https://user-images.githubusercontent.com/24260856/124215264-52805080-db11-11eb-8b04-0bf90bff1445.jpg)
![Screenshot_2021-07-02_08-41-36](https://user-images.githubusercontent.com/24260856/124215274-5613d780-db11-11eb-91ff-0be0fd1a0bf2.png)

**Comment by C47D** [comment]
You just answered it, the display needs the DC signal to work, you can't use it without it.

**Comment by Rajssss** [comment]
> You just answered it, the display needs the DC signal to work, you can't use it without it.

I don't get it. Are you saying that the display I have can not work without the DC pin? exactly what I am trying to achieve?
And is it because the DC pin is exposed in the module?

**Comment by C47D** [comment]
> I don't get it. Are you saying that the display I have can not work without the DC pin?

Yes, I think so, your project works with the DC signal and it doesn't work without it.

> And is it because the DC pin is exposed in the module?

Yes.

The only displays I **think** work without DC signal are the I2C based ones.

**Comment by Rajssss** [comment]
> > I don't get it. Are you saying that the display I have can not work without the DC pin?
> 
> Yes, I think so, your project works with the DC signal and it doesn't work without it.
> 
> > And is it because the DC pin is exposed in the module?
> 
> Yes.
> 
> The only displays I **think** work without DC signal are the I2C based ones.

Unfortunately, I don't have one. maybe in future someone will test it.
I have found a thread on a display without DC pin exposed, using 9Bit SPI but for ILI9341, can you take a look?
